packages/filter/src/CompactFilter.tsx

```diff
@@ -52,14 +52,14 @@ const PureCompactFilter: React.FC<IFilterComponentProps> = ({
 
 
   return (
-    <RootPanel>
+    <RootPanel theme={ themeRef.current }>
```

Comment:
Maybe we just wrap everything in the ThemeProvider and pass it the theme, instead of passing it to each component?